src/AppInstallerCLICore/ExecutionArgs.h

```diff
@@ -86,6 +86,10 @@ namespace AppInstaller::CLI::Execution
             CustomHeader, // Optional Rest source header
             AcceptSourceAgreements, // Accept all source agreements
             IncludeUnknown, // Used in Upgrade command to allow upgrades of packages with unknown versions
+            Rename, // Renames the file of the executable. Only applies to the portable installerType

```

Comment:
>Rename [](http://example.com/codeflow?start=12&length=6)

nit: some of these might better fit in Instal behavior or Uninstallbehavior rather than Other category

Reply:
Moved these arguments around to their appropriate categories